api/apps/api/src/modules/legacy-project-import/application/start-legacy-project-import.handler.ts

```diff
@@ -45,6 +45,7 @@ export class StartLegacyProjectImportHandler
       const scenario = await this.scenarioRepo.save({
         name,
         projectId: project.id,
+        solutions_are_locked: true,
```

Comment:
As we talked, `soulutions_are_locked` value will be provided by the user. We should add a parameter to `StartLegacyProjectImport` command